### DISCUSSION TITLE
【项目组队】sui-dapp-scaffold | Bounty：100U

### DISCUSSION THREAD

**Discussion by ShirleneLiu** [creatorsand/.github#35] [Ideas]
项目名称：Sui-dapp-scaffold

项目描述：
1.hook 语法的react +
2. mui
3. 集成官方提供的 https://github.com/MystenLabs/sui/tree/main/sdk/wallet-adapter
4. typescript
5.参考 https://github.com/solana-labs/dapp-scaffold

项目发起人：uvd

其他角色需求：前端

有无Bounty：100U

原链：https://github.com/movefuns/co-learn/discussions/6

**Comment by uvd** [comment]
需要把调用钱包sdk 对接好一些常用功能

**Reply by ClaraHayesDev** [reply]
Claimed
